src/components/Visualizations/Table/Styled.jsx

```diff
@@ -10,6 +10,7 @@ export const StyledContainer = styled.div`
 export const KDAContainer = StyledContainer.extend`
   width: calc(300% + 50px);
   left: -10px;
+  pointer-events none;
```

Comment:
doesn't this need a colon?

Reply:
lol I actually forgot the colons but it's working without colons too for some reason